Signal/src/ViewControllers/CallViewController.swift

```diff
@@ -450,18 +451,18 @@ class CallViewController: UIViewController, CallObserver, CallServiceObserver, R
             // Dark blurred background.
             blurView.autoPinEdgesToSuperviewEdges()
 
-            localVideoView.autoPinEdge(toSuperviewEdge:.right, withInset:videoPreviewHMargin)
+            localVideoView.autoPinTrailingToSuperView(withMargin: videoPreviewHMargin)
             localVideoView.autoPinEdge(toSuperviewEdge:.top, withInset:topMargin)
             let localVideoSize = ScaleFromIPhone5To7Plus(80, 100)
             localVideoView.autoSetDimension(.width, toSize:localVideoSize)
             localVideoView.autoSetDimension(.height, toSize:localVideoSize)
 
             contactNameLabel.autoPinEdge(toSuperviewEdge:.top, withInset:topMargin)
-            contactNameLabel.autoPinEdge(toSuperviewEdge:.left, withInset:contactHMargin)
```

Comment:
Caller name and duration label seem a little too close to the left edge now. (in LTR)

![screen shot 2017-07-20 at 11 59 26 am](https://user-images.githubusercontent.com/217057/28427062-0374ad5a-6d43-11e7-88f7-654edf371786.png)